Resources/config/menu.xml

```diff
@@ -17,6 +17,7 @@
         <parameter key="knp_menu.renderer.list.options" type="collection"></parameter>
         <parameter key="knp_menu.listener.voters.class">Knp\Bundle\MenuBundle\EventListener\VoterInitializerListener</parameter>
         <parameter key="knp_menu.voter.router.class">Knp\Menu\Matcher\Voter\RouteVoter</parameter>
+        <parameter key="knp_menu.manipulator.class">Knp\Menu\Util\MenuManipulator</parameter>
```

Comment:
please don't introduce new class parameters. Using them is discouraged in Symfony